packages/dappmanager/src/modules/chains/getChainDriverName.ts

```diff
@@ -17,6 +17,8 @@ const knownChains: { [dnpName: string]: ChainDriver } = {
   "kovan.dnp.dappnode.eth": "ethereum",
   "bitcoin.dnp.dappnode.eth": "bitcoin",
   "monero.dnp.dappnode.eth": "monero",
-  "prysm.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm",
-  "prysm-prater.dnp.dappnode.eth": "ethereum2-beacon-chain-prysm"
+  "prysm.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "prysm-prater.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "lighthouse-prater.dnp.dappnode.eth": "ethereum-beacon-chain",
+  "teku-prater.dnp.dappnode.eth": "ethereum-beacon-chain"
```

Comment:
Can you delete this file in another PR? I'm tired of everyone commiting to this file :rofl:

Reply:
Done in https://github.com/dappnode/DNP_DAPPMANAGER/pull/1021